Loops/Problem10/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Problem10
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int n = int.Parse(Console.ReadLine());
+
+            int[,] matrix = new int[n, n];
+
+            int counter = 1;
+
+            for (int row = 0; row < n; row++)
```

Comment:
Why need to create and populate matrix and have 4 cycles couldn't be done with less cycles?